docs/basics/1d_hist.rst

```diff
@@ -218,3 +218,99 @@ To compare two histograms, using the histograms defined above:
    :alt: Simple hist
    :width: 500
 
+
+Comparing two histograms (advanced)
```

Comment:
Should not be in Basic 1D examples, but in a new section called Advanced 1D examples